### PR TITLE
Add timeline placeholder content

### PR DESCRIPTION
**Pull request by DilwoarH**
## What
Add a timeline of recent changes to the landing page to communicate to users what rules are changing or have recently changed, and what they need to do differently from what date.

## Why
We know that users are struggling to find out what has recently changed, and this impacts how likely they are to follow the rules, and how confident they feel that the landing page is up to date and relevant.

We hope that this timeline element will help users:

- understand what's changed
- feel confident that the information is up to date
- ultimately comply with new rules and help prevent the spread of covid

https://trello.com/c/XUyYJ8lt/813-24-sept-landing-page-add-timeline-of-changes-element

### CODE REVIEW

File: content/coronavirus_landing_page.yml [hunk 1]
```diff
@@ -60,6 +60,18 @@ content:
       text: "Find support"
   explainer_title:
   explainer_text:
+  timeline:
+    heading: Recent and upcoming changes
+    list:
+      - heading: 1 October
```
Comment (leenagupte) [inline content/coronavirus_landing_page.yml:66]: Are we expecting content designers to add dates like this? Or will we accept a date in any UK format and then apply formatting?

**Review by leenagupte — APPROVED** [review]
A couple of questions about dates that can be answered later.

File: content/coronavirus_landing_page.yml (hunk 1 above)
Comment (leenagupte) [reply content/coronavirus_landing_page.yml:66]: Also, should this be "effective date" rather than "heading"?

Comment (DilwoarH) [reply content/coronavirus_landing_page.yml:66]: My plan is to just take whatever heading they set and show it without applying any formatting - Let the content designer decide on format

**Pull request merged**